dask/dataframe/core.py

```diff
@@ -3038,6 +3074,33 @@ def drop_columns(df, columns, dtype):
     return df
 
 
+def idxmaxmin_chunk(x, **kwargs):
+    fn = kwargs['fn']
+    axis = kwargs['axis']
+    idx = getattr(x, fn)(axis=axis)
+    minmax = 'max' if fn == 'idxmax' else 'min'
+    value = getattr(x, minmax)(axis=axis)
+    n = len(x)
+    if isinstance(idx, pd.Series):
+        return pd.DataFrame({'idx': idx, 'value': value, 'n': [n] * len(idx)}).T
+    return pd.DataFrame({'idx': [idx], 'value': [value], 'n': [n]}).T
+
+def idxmaxmin_row(x, fn=None):
+    idx = x.ix['idx'].reset_index(drop=True) # .astype(int)
+    try:
+        idx = idx.astype(int)
+    except:
+        pass
```

Comment:
Is this always correct?  Why might it fail?  Are these index values or index locations?  If values then is this dangerous, for example if we have a floating point index?  If locations then isn't the try-except unnecessary?